packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -200,19 +207,53 @@ export class TeamsBot implements Plugin {
       return await runWithExceptionCatching(
         context,
         async () => {
-          if (isVSProject(context.projectSettings) || context.answers?.platform === Platform.VS) {
-            const res = new QTreeNode(createHostTypeTriggerQuestionForVS());
-            res.condition = showNotificationTriggerCondition;
-            return ok(res);
-          } else if (isBotNotificationEnabled()) {
-            const res = new QTreeNode({
-              type: "group",
-            });
-            res.addChild(new QTreeNode(createHostTypeTriggerQuestion(context.answers?.platform)));
-            res.condition = showNotificationTriggerCondition;
-            return ok(res);
+          if (isCLIDotNetEnabled()) {
```

Comment:
Nit: For quick implementation, it's okay. But I sensed that this piece of code is worth refactoring to make it simple and concise.